common/changes/office-ui-fabric-react/stanleyy-fillin_2018-02-07-00-36.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Add multiSelect capability for ComboBox",
```

Comment:
I don't think this changefile is needed since the one below seems to be the correct file. Can you remove this one?

Reply:
You are right. Will delete.